localflavor/ma/forms.py

```diff
@@ -74,3 +74,26 @@ class MARegionField(CharField):
     def __init__(self, **kwargs):
         kwargs.setdefault('label', _('Select Region'))
         super().__init__(**kwargs)
+
+
+class MACinNumberField(RegexField):
+    """
+        CIN number: (Numéro de la Carte D'Identité Nationale) The CIN represents the ID of a Moroccan citizen.
+
+         - It is an 8-max-length string that starts with one or two Latin letters followed by digits,
+           with the first digit not being zero.
+
+         - as implemented in the official government site "https://www.cnie.ma/"
+        .. versionadded:: 4.1
+    """
+
+    default_error_messages = {
+        'invalid': _('Enter a valid Moroccan CIN number.'),
+    }
+    cin_pattern = r'^[A-Za-z]{1,2}[1-9]\d{0,6}$'
```

Comment:
I think \d in python3 include unicode digits. Try this in a python console:

```
import re; re.match(r'\d', '٠١٢٣٤٥٦٧٨٩')
```

I think it should be this:
```suggestion
    cin_pattern = r'^[A-Za-z]{1,2}[1-9][0-9]{0,6}$'
```

Reply:
Thanks for pointing that out ! , I will update the code accordingly